src/DiffEq_Hamiltonians.jl

```diff
@@ -427,7 +427,7 @@ end
 function add_partial_derivative!(∂m∂t, m, mfun, p::Tuple{Real,Real,Any,Any,Any,Any,Any,Any,Real,Real,Any}, t, grad_type::grad_B1)
     ω1, B1, ω0, m0s, R1f, R2f, Rx, R1s, T2s, TRF, dG_o_dT2s_x_T2s = p
 
-    f_PSD(τ) = quadgk(ct -> 1 / abs(1 - 3 * ct^2) * (4 / τ / abs(1 - 3 * ct^2) * (exp(- τ^2 / 8 * (1 - 3 * ct^2)^2) - 1) + sqrt(2π) * erf(τ / 2 / sqrt(2) * abs(1 - 3 * ct^2))), 0, 1, order=7)[1]
+    f_PSD(τ) = quadgk(ct -> 1 / abs(1 - 3 * ct^2) * (4 / τ / abs(1 - 3 * ct^2) * (exp(- τ^2 / 8 * (1 - 3 * ct^2)^2) - 1) + √(2π) * erf(τ / 2√2 * abs(1 - 3 * ct^2))), 0, 1, order=7)[1]
```

Comment:
This line seems to cause an error in Julia 1.6 – no idea why
```Julia
ERROR: LoadError: LoadError: syntax: missing comma or ) in argument list
Stacktrace:
  [1] top-level scope
    @ ~/work/MRIgeneralizedBloch.jl/MRIgeneralizedBloch.jl/src/DiffEq_Hamiltonians.jl:430
    ```

Reply:
Of course! Oh, the square root operator `√()` seems to not be valid Julia 1.6. I can revert those changes.